packages/docs/docs/how-tos/get-status.md

```diff
@@ -0,0 +1,100 @@
+
+On the [Account page](https://web3.storage/account) of the Web3.Storage site, you can see a list of files that you've uploaded to Web3.Storage.
+
+![Screenshot of Web3.Storage account page, showing a table of uploaded files](./images/account-page-files.png)
+
+The table on the account page has two columns related to the current status of the data storage process, [Status](#pin-status) and [Storage Providers](#storage-providers).
+
+The information in the account page file list is also available using the [JavaScript client's `check` method][client-js-check], and via the [HTTP API][client-http] by sending a `GET` request to the `/check/{cid}` endpoint.
+
+## Status
+
+When you upload data to Web3.Storage, it is ["pinned"][ipfs-docs-pinning] to several [IPFS clusters][ipfs-cluster], which are groups of machines that coordinate to provide content-addressed data to the [IPFS](https://ipfs.io) peer-to-peer network.
+
+The **Status** column shows the status of the pinning operation. There are three possible values for the Status column: `Queuing`, `Pinned`, and `Failed`.
+
+The `Queuing` status indicates that a request to pin the data has been submitted to a cluster node, and the node has added the request to a queue for pinning. Depending on how busy the cluster is, an upload may stay in the `Queuing` state for some time before transitioning to `Pinned` or `Failed`.
+
+Note that you can [retrieve your data using IPFS][howto-retrieve] even while the status is `Queuing`. You can verify this by retrieving your data from an HTTP gateway. If you've uploaded a directory, try retrieving a sample of the files within the directory.
+
+A `Pinned` status means that the cluster has successfully pinned the data.
+
+A `Failed` status indicates that an operation failed while attempting to pin the data. Note that a `Failed` status may be temporary, as the cluster will continually retry `Failed` pins in the background and will update the status to `Pinned` if a retry attempt is successful.
+
+In addition to automatic retries by the cluster, a `Failed` pin will change to `Pinned` if the same data is uploaded successfully in a subsequent request. As long as the CID is the same for both uploads, the original upload's status will be marked as `Pinned` after a small delay.
+
+There are several reasons why an upload may end up in the `Failed` state.
+
+If you are able to [retrieve the data from IPFS][howto-retrieve], the cluster is likely waiting to retry a previous attempt and will soon pin the data.
+
+If you uploaded your data directly to the Web3.Storage service using a [client library][client-js], [the HTTP API][client-http], or the [Upload files button on the website account page][upload-page], a `Failed` status may indicate that one or more chunks of data was not recieved by the service. This may happen when uploading large files or directories containing many files and may be resolved by retrying the upload.
```

Comment:
```suggestion
If you uploaded your data directly to the Web3.Storage service using a [client library][client-js], [the HTTP API][client-http], or the [Upload files button on the website account page][upload-page], a `Failed` status may indicate that one or more chunks of data was not received by the service. This may happen when uploading large files or directories containing many files and may be resolved by retrying the upload.
```